actors/builtin/market/market_balances.go

```diff
@@ -0,0 +1,115 @@
+package market
+
```

Comment:
@anorth @ZenGround0 Except for `lockBalanceOrAbort `, all other functions have simply been moved here for `market_state.go`